domene/src/main/java/no/nav/foreldrepenger/journalføring/oppgave/domene/NyOppgave.java

```diff
@@ -1,8 +1,10 @@
 package no.nav.foreldrepenger.journalføring.oppgave.domene;
 
+import no.nav.foreldrepenger.fordel.kodeverdi.BehandlingTema;
 import no.nav.foreldrepenger.journalføring.domene.JournalpostId;
+import no.nav.foreldrepenger.journalføring.oppgave.lager.BrukerId;
```

Comment:
Er ikke helt fan av å blande domene med lager - vi bør i utgangspunkt opprette felles DB objekter og gjenbruke den på tvers av apper for persistering (dette vil gjøre database laget mye enklere å vedlikeholde uten fare at man ødelegger ting i domene - og motsatt. 

Tror på sikt bør vi kunne flytte en del felles ting til f.eks felles-db dette vil føre til at man får det samme oppførsel i mange apper mot databasen. 

Det samme bør man tenke for Dtoer - kanskje fp-kontrakter er en bra kandidat for å hoste slike objekter og så kan man tenke over felles domene objekter og mappere.

Reply:
Nå ligger entitetene allerede i samme modul "domene" så ting er blandet i en liten app. 
Gjorde dette for å redusere String-bruken. Klassen burde hete AktørId når det forventes.

Strengt tatt kan vi lage en mengde verdi-typer med validering av pattern i en rot-modul felles - domenetyper.
Så kan felles-db extende eller embedde slike i Embeddable-klasser, mens Dtos kan lage varianter med Validation.
Men det er en egen aktivitet som kan settes opp i Teknisk-EPOS.